Makefile

```diff
@@ -37,7 +37,7 @@ lint:
 	golangci-lint run ${FIX} ./...
 
 lint-markdown:
```

Comment:
Oh. I'm sorry i just noticed we just use the action for our markdown lint ci job. Yeah if you will just remove the `${FiX}` now. My apologies.